--- netcdf-c-org/CMakeLists.txt	2020-06-19 10:13:15.584452298 +0200
+++ netcdf-c/CMakeLists.txt	2020-06-19 11:41:10.440452298 +0200
@@ -11,6 +11,7 @@
 
 #Project Name
 project(netCDF C)
+#        conan_basic_setup()
 set(PACKAGE "netCDF" CACHE STRING "")
 #Project Version
 SET(netCDF_VERSION_MAJOR 4)
@@ -50,10 +51,9 @@
     SET(GLOBAL PROPERTY USE_FOLDERS ON)
 ENDIF()
 
-
 #Add custom CMake Module
-SET (CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules/"
-    CACHE INTERNAL "Location of our custom CMake modules.")
+SET (CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules/")
+include(${CMAKE_BINARY_DIR}/conan_paths.cmake)
 
 # auto-configure style checks, other CMake modules.
 INCLUDE (${CMAKE_ROOT}/Modules/CheckLibraryExists.cmake)
@@ -629,9 +629,10 @@
     MESSAGE(STATUS "Using HDF5_HL LIbrary: ${HDF5_HL_LIB}")
   ELSE()
     IF(MSVC)
-      FIND_PACKAGE(HDF5 COMPONENTS C HL NO_MODULE REQUIRED)
+      FIND_PACKAGE(HDF5 COMPONENTS C HL MODULE REQUIRED)
     ELSE()
       FIND_PACKAGE(HDF5 COMPONENTS C HL REQUIRED)
+      SET(HDF5_hdf5_LIBRARY HDF5::HDF5)
     ENDIF()
   ENDIF()
 
@@ -664,11 +665,11 @@
 
   # Assert HDF5 version.
   SET(HDF5_VERSION_REQUIRED 1.8.6)
-  execute_process(COMMAND sh -c "h5cc -showconfig | grep -i \"HDF5 version\" | sed 's/^.*[^0-9]\\([0-9]*\\.[0-9]*\\.[0-9]*\\).*$/\\1/'"
-                  OUTPUT_VARIABLE HDF5_VERSION
-		  ERROR_QUIET
-                  )
-  
+  #execute_process(COMMAND sh -c "h5cc -showconfig | grep -i \"HDF5 version\" | sed 's/^.*[^0-9]\\([0-9]*\\.[0-9]*\\.[0-9]*\\).*$/\\1/'"
+  #                OUTPUT_VARIABLE HDF5_VERSION
+  # 	  ERROR_QUIET
+  #                )
+
   # It cannot be assumed that libhdf5.settings exists.  If it doesn't,
   # the call to h5cc will have failed.  At this point we will have to
   # trust the user that they have configured their system properly.
@@ -716,7 +717,7 @@
 
   # Check for ZLib, but only if using HDF5.
   FIND_PACKAGE(ZLIB)
-  IF(NOT ZLIB_LIBRARY)
+  IF(NOT ZLIB_FOUND)
   MESSAGE(FATAL_ERROR "HDF5 Support specified, cannot find ZLib.")
   ENDIF()
   SET(USE_ZLIB ON)
