From aeeeab96a03a4bff3f64cd036df9b4320ef66832 Mon Sep 17 00:00:00 2001
From: Joakim Haugen <joakim.haugen@sintef.no>
Date: Mon, 10 Apr 2023 12:50:01 +0200
Subject: [PATCH] Tweak CMakeLists

---
 CMakeLists.txt | 16 ++++++----------
 1 file changed, 6 insertions(+), 10 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 34f2c70b..03c85acb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,7 @@
 ##################################
 
 #Minimum required CMake Version
-cmake_minimum_required(VERSION 2.8.8)
+cmake_minimum_required(VERSION 2.8.12)
 
 #Project Name
 project(netCDF C)
@@ -52,8 +52,7 @@ ENDIF()
 
 
 #Add custom CMake Module
-SET (CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules/"
-    CACHE INTERNAL "Location of our custom CMake modules.")
+SET (CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules/")
 
 # auto-configure style checks, other CMake modules.
 INCLUDE (${CMAKE_ROOT}/Modules/CheckLibraryExists.cmake)
@@ -629,9 +628,11 @@ IF (USE_HDF5 OR ENABLE_NETCDF_4)
     MESSAGE(STATUS "Using HDF5_HL LIbrary: ${HDF5_HL_LIB}")
   ELSE()
     IF(MSVC)
-      FIND_PACKAGE(HDF5 COMPONENTS C HL NO_MODULE REQUIRED)
+      FIND_PACKAGE(HDF5 COMPONENTS C HL REQUIRED)
+      SET(HDF5_hdf5_LIBRARY HDF5::HDF5)
     ELSE()
       FIND_PACKAGE(HDF5 COMPONENTS C HL REQUIRED)
+      SET(HDF5_hdf5_LIBRARY HDF5::HDF5)
     ENDIF()
   ENDIF()
 
@@ -664,11 +665,6 @@ IF (USE_HDF5 OR ENABLE_NETCDF_4)
 
   # Assert HDF5 version.
   SET(HDF5_VERSION_REQUIRED 1.8.6)
-  execute_process(COMMAND sh -c "h5cc -showconfig | grep -i \"HDF5 version\" | sed 's/^.*[^0-9]\\([0-9]*\\.[0-9]*\\.[0-9]*\\).*$/\\1/'"
-                  OUTPUT_VARIABLE HDF5_VERSION
-		  ERROR_QUIET
-                  )
-  
   # It cannot be assumed that libhdf5.settings exists.  If it doesn't,
   # the call to h5cc will have failed.  At this point we will have to
   # trust the user that they have configured their system properly.
@@ -716,7 +712,7 @@ IF (USE_HDF5 OR ENABLE_NETCDF_4)
 
   # Check for ZLib, but only if using HDF5.
   FIND_PACKAGE(ZLIB)
-  IF(NOT ZLIB_LIBRARY)
+  IF(NOT ZLIB_FOUND)
   MESSAGE(FATAL_ERROR "HDF5 Support specified, cannot find ZLib.")
   ENDIF()
   SET(USE_ZLIB ON)
-- 
2.33.0.windows.2

