--- netcdf-c/config.h.in.cmake	2020-06-18 13:45:29.734651800 +0200
+++ netcdf-c/config.h.in.cmake.new	2020-06-19 09:33:44.404708800 +0200
@@ -30,7 +30,9 @@
 #ifdef _WIN32
 	#ifndef strcasecmp
   		#define strcasecmp _stricmp
-		#define snprintf _snprintf
+		#if _MSC_VER < 1900
+			#define snprintf _snprintf
+		#endif
   	#endif
 #endif
 
@@ -38,7 +40,9 @@
   #define strdup _strdup
   #define fdopen _fdopen
   #define write _write
-  #define snprintf _snprintf
+  #if _MSC_VER < 1900
+    #define snprintf _snprintf
+  #endif	
   #define strtoll _strtoi64
 #endif
 
